examples/platforms/posix/radio.c

```diff
@@ -177,7 +177,7 @@ static inline bool isDataRequestAndHasFramePending(const uint8_t *aFrame)
     const uint8_t *cur = aFrame;
     uint8_t        securityControl;
     bool           isDataRequest   = false;
-    bool           hasFramePending = false;
+    bool           hasFramePending = true;
```

Comment:
`bool hasFramePending = !sSrcMatchEnabled`?

Reply:
Isn't ` = true` simpler? There is really no actual difference.